*** qdmail.php.org
--- qdmail.php
***************
*** 3820,3826 ****
  		$view = & new $this->Controller->view( $this->Controller , false );
  		$view->layout = $this->layout;
  		$mess = null;
! 		$content = $view->renderElement( $this->view_dir . DS . $type . DS . $this->template , array('content' => $content ) , true );
  		if( 1.2 > (float) substr(Configure::version(),0,3) ){
  			$view->subDir = $this->layout_dir . DS . $type . DS ;
  		}else{
--- 3820,3830 ----
  		$view = & new $this->Controller->view( $this->Controller , false );
  		$view->layout = $this->layout;
  		$mess = null;
!         if (version_compare(Configure::version(), '1.3.0') < 0) {
!             $content = $view->renderElement( $this->view_dir . DS . $type . DS . $this->template , array('content' => $content ) , true );
!         } else {
!             $content = $view->element( $this->view_dir . DS . $type . DS . $this->template , array('content' => $content ) , true );
!         }
  		if( 1.2 > (float) substr(Configure::version(),0,3) ){
  			$view->subDir = $this->layout_dir . DS . $type . DS ;
  		}else{
